sql-parser/src/main/java/io/crate/sql/SqlFormatter.java

```diff
@@ -421,6 +421,12 @@ public Void visitPrimaryKeyColumnConstraint(PrimaryKeyColumnConstraint node, Int
         }
 
         @Override
+        public Void visitNotNullColumnConstraint(NotNullColumnConstraint node, Integer indent) {
+            builder.append("NOT NULL");
```

Comment:
do we have a test for this, is whitespace around "not null"  if more than one constraint